test/built-ins/Temporal/PlainDateTime/prototype/withPlainDate/unrecognized-properties.js

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
```

Comment:
It's more convenient for keeping track of what coverage there is, if you can pattern similar tests in different methods similarly, and also name them similarly.

For example, the tests for with() have the empty object, objects with only unrecognized properties, and objects with both recognized and unrecognized properties, as three different tests (and I suggested combining the former two), whereas this file combines the latter two, and withPlainTime() combines all three.

Reply:
makes sense -- I've tried to clean things up as suggested